fsw/cfe-core/unit-test/sb_UT.h

```diff
@@ -3242,7 +3285,10 @@ void Test_CFE_SB_SetGetMsgId(void);
 ** \sa #UT_GetActualPktLenField, #UT_Report
 **
 ******************************************************************************/
-void Test_CFE_SB_SetGetUserDataLength(void);
+void Test_CFE_SB_SetGetUserDataLength_Cmd(void);
```

Comment:
dumb question - why are individual test fn's defined in the .h file? It's not like anything else is calling them...